modules/dynamic-plug-in-development.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-get-started.adoc
+
+:_content-type: PROCEDURE
+[id="dynamic-plugin-development_{context}"]
+= Dynamic plug-in development
+
+You can run the plug-in using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
```

Comment:
Do we need this? Seems obvious to me. Maybe I am mistaken:

> The {product-title} web console runs in a container connected to the cluster you have logged into.

Reply:
I don't think it's obvious? We're not talking about the console that runs as part of your OpenShift cluster, but one you're running locally. Maybe we need to make that more clear.